modules/candidate_parameters/test/TestPlan.md

```diff
@@ -67,4 +67,18 @@
         * Error: date cannot be later than today
     * Consent to Study = No (changing/updating an already existing 'Yes' consent); Date (Withdrawal) of Consent = valid random date; Confirmation (Withdrawal) Date of Consent = same random date
         * No error
-        * Make sure they update properly in the front-end and backend
\ No newline at end of file
+        * Make sure they update properly in the front-end and backend
+
+### Date of Birth Tab
+32. Check that date of birth can only be amended if user has candidate_dob_edit permission.
+33. Change the date of birth and try to save it. Check that it is updated on front-end and back-end.
+34. Check that entering a date later than today gives a data entry error.
+35. Check that the date stored matches the _DOB Format_ specified in the configuration module. 
+
+### Date of Death Tab
+36. Check that date of death can only be set if user has candidate_dod_edit permission.
```

Comment:
does the tab show to the user if they dont have the permission ?

Reply:
yes, however the field is disabled.